src/main/resources/admin/tools/content-type-manager/components/component-file-handler.html

```diff
@@ -206,12 +211,7 @@
 
             Swal.fire({
               icon: "success",
-              showConfirmButton: false,
-              timer: 1500,
-            });
-          } else {
-            Swal.fire({
-              icon: "error",
+              text: this.warnings.save,
```

Comment:
Success icon. Warning text 😕 ? 
Change warning to messages or popupText? Open for other suggestions

Reply:
@poi33 yep, makes sense! What do you think about "dialogMessages" ?

Reply:
On point and descriptive. 👍 go for it.